pdc/apps/auth/views.py

```diff
@@ -231,7 +237,9 @@ def get_api_perms(request):
     for obj in models.ResourcePermission.objects.all():
         name = URL_ARG_RE.sub(r'{\1}', obj.resource.name)
         url = ret[name]
-        members = get_users_and_groups(obj)
+        members = get_members(obj)
+        if read_permission_for_all() and obj.permission.name == 'read':
+            members = ['@all']
         perms.setdefault(name, OrderedDict()).setdefault(obj.permission.name, set()).update(members)
```

Comment:
About part is not related to `Create a new API of api/resource-permission`. Normally in a separate commit is better.